gemfiles/rails_4.2_stable.gemfile

```diff
@@ -2,7 +2,7 @@
 
 source "https://rubygems.org"
 
-gem "bundler", "< 2"
```

Comment:
for consistency, you should delete this file

Reply:
Not gonna delete for now because I'm leaving this in until Rails 4.2 is officially EOL